manifests/istio-control/istio-discovery/templates/deployment.yaml

```diff
@@ -83,8 +83,12 @@ spec:
           - "-a"
           - {{ .Release.Namespace }}
 {{- end }}
-          - --secureGrpcAddr
-          - ""
+
+{{- if and .Values.global.controlPlaneSecurityEnabled .Values.global.istiod.enabled }}
```

Comment:
I would not mix controlPlaneSecurityEnabled here.

Just istiod.

Reply:
Are you suggesting `if istiod { :15011 }`? Do we not want the option to not have the 15011 listener?